config/settings/test.yml

```diff
@@ -26,18 +26,15 @@ saml_ssoe:
   cert_path: spec/support/certificates/ruby-saml.crt
   key_path: spec/support/certificates/ruby-saml.key
 
-
 edu:
   prefill: true
   sftp:
-    relative_path: 'spool_files'
-
+    relative_path: "spool_files"
```

Comment:
Looks like this may have been added by mistake. rubocop prefers single quotes

Reply:
Oh thanks, I will fix this now!